scheduler/src/cook/mesos.clj

```diff
@@ -255,6 +258,15 @@
                                                                               :pending-jobs-atom mesos-pending-jobs-atom
                                                                               :trigger-chan rebalancer-trigger-chan
                                                                               :view-incubating-offers view-incubating-offers})
+                                    (when (seq optimizer-config)
+                                      (cook.mesos.optimizer/start-optimizer-cycles! (fn get-queue []
+                                                                                      ;; TODO Use filters scheduler uses
+                                                                                      @mesos-pending-jobs-atom)
+                                                                                    (fn get-running []
+                                                                                      (cook.mesos.util/get-running-task-ents (d/db mesos-datomic-conn)))
```

Comment:
What is the impact on datomic performance for making this query with respect to other components in Cook given it is invoked every 10 seconds??

Reply:
I'm not sure how to measure it but my expectation is very little. That query is around 100ms in prod at the moment and it is single threaded.